### PR TITLE
chore(deps): update deps

### PR DESCRIPTION
**Pull request by jrandolf-zz**
Updates development and standard dependencies. Also replaces `node-fetch` with `cross-fetch` for compatibility.

### CODE REVIEW

File: src/common/Page.ts [hunk 1]
```diff
@@ -1795,12 +1795,12 @@ export class Page extends EventEmitter {
    *   more than 2 network connections for at least `500` ms.
    */
   async reload(options?: WaitForOptions): Promise<HTTPResponse | null> {
-    const result = await Promise.all<HTTPResponse, void>([
+    const result = await Promise.all<HTTPResponse | void>([
```
Comment (OrKoN) [inline src/common/Page.ts]: was it intended to be `Promise.all<[HTTPResponse, void]>` instead? (if that compiles)

File: src/common/fetch.ts [hunk 2]
```diff
@@ -1,22 +0,0 @@
...
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-import { isNode } from '../environment.js';
-
-/* Use the global version if we're in the browser, else load the node-fetch module. */
-export const getFetch = async (): Promise<typeof fetch> => {
```
Comment (OrKoN) [inline src/common/fetch.ts:20]: let's keep conditional imports here as in DevTools (browser version) we cannot have any dependencies as we don't use node module resolution.

File: src/common/Page.ts (hunk 1 above)
Comment (jrandolf-zz) [reply src/common/Page.ts]: This will no longer compile in new versions of typescript.

File: compat/cjs/compat.ts [hunk 3]
```diff
@@ -1,3 +1,19 @@
+/**
+ * Copyright 2020 Google Inc. All rights reserved.
```
Comment (OrKoN) [inline compat/cjs/compat.ts]: 2022 for new files

**Review by OrKoN — APPROVED** [review]
LGTM thanks!

**Pull request merged**